src/wechaty/accessory.go

```diff
@@ -4,6 +4,10 @@ import (
 	wechatyPuppet "github.com/wechaty/go-wechaty/src/wechaty-puppet"
 )
 
-type Accessory struct {
-	Puppet wechatyPuppet.Puppet // wechat-puppet 的 Puppet 接口
+type Accessory interface {
```

Comment:
Q2: I also have some doubts about the definition of Accessory, I am not completely sure.